src/lib.rs

```diff
@@ -952,7 +951,7 @@ impl Drop for Rav1dContext {
                         &mut (*f).tile_thread.lowest_pixel_mem as *mut *mut [[c_int; 2]; 7]
                             as *mut c_void,
                     );
-                    freep(&mut (*f).frame_thread.b as *mut *mut Av1Block as *mut c_void);
+                    mem::take(&mut (*f).frame_thread.b); // TODO: remove when context is owned
```

Comment:
```suggestion
                    let _ = mem::take(&mut (*f).frame_thread.b); // TODO: remove when context is owned
```

Not strictly needed but we've been writing it like this for `take`s that should be done automatically in `impl Drop`s, which helps with searchability.